src/Symfony/Component/Security/Http/Authenticator/AccessToken/BodyAccessTokenExtractor.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator\AccessToken;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Extracts a token from the body request.
+ *
+ * @author Florent Morselli <florent.morselli@spomky-labs.com>
+ *
+ * @final
+ */
+class BodyAccessTokenExtractor implements AccessTokenExtractorInterface
+{
+    public const PARAMETER = 'access_token';
```

Comment:
should probably be named `DEFAULT_PARAMETER`, as that's not always the parameter being used

Reply:
OK. The constant was renamed.